src/app/stripeObject.js

```diff
@@ -0,0 +1,28 @@
+import { randomInteger } from './randomThings'
+
+const stripeObject = (id, colors, settings) => {
+  let {
+    position,
+    direction,
+    opacity,
+    width
+  } = settings
+
+  const color = colors[randomInteger(0, colors.length - 1)]
+
+  position = randomInteger(position.min, position.max)
```

Comment:
it threw me for a loop that position, etc first are objects with ranges but then are transformed into percentages, but still the same variable.

Reply:
Those variables were _variable as hell!_ I created new variables for the random values to eliminate confusion.